examples/ipython-demo.ipynb

```diff
@@ -1,202 +1,203 @@
 {
- "metadata": {
-  "name": "",
-  "signature": "sha256:81f3deed7cdc26b0fc756b3ee1eb6e8f9b1be96304ddfc6ff484d223c2b8a942"
- },
- "nbformat": 3,
- "nbformat_minor": 0,
- "worksheets": [
+ "cells": [
```

Comment:
Modified a single `np.random.rand` here and it updated the whole notebook :confounded: Let me know if you want this reverted or something?